packages/minimongo/minimongo.js

```diff
@@ -180,6 +180,42 @@ LocalCollection.Cursor.prototype.forEach = function (callback, thisArg) {
   });
 };
 
+LocalCollection.Cursor.prototype[Symbol.iterator] = function () {
+  var self = this;
```

Comment:
As I said in already mentioned issue, I'm planning to submit a PR with making `minimongo` _more ES5/6_.